nano/core_test/conflicts.cpp

```diff
@@ -71,6 +71,79 @@ TEST (conflicts, DISABLED_add_two)
 	ASSERT_EQ (2, node1.active.size ());
 }
 
+TEST (conflicts, add_two)
```

Comment:
Can you add a comment in english that explains what this test case does?
Also a comment about the subtle but important fact that the node cannot vore because it does not have a signing key.
Like the annotations here:
https://github.com/nanocurrency/nano-node/issues/3535#issuecomment-959827386

Reply:
Sounds good, I'll add a couple comments.

Reply:
I think this duplicates https://github.com/nanocurrency/nano-node/issues/3535.